### DISCUSSION TITLE
Сhange the color on the timeline for a task

### DISCUSSION THREAD

**Discussion by maerty1** [Leantime/leantime#1933] [Ideas]
Hello
Make it possible to change the timeline color for a task.
This feature v2.3.27 is only available for milestones, but I would really like to have this feature for tasks and subtasks.
This may not be the most important idea, but thanks.

**Comment by FFaranna** [comment]
I need this, too - it gets super messy and confusing with lots of tasks.

**Comment by marcelfolaron** [comment]
Are you looking to set the color per individual task  or at a higher level
(say task types or tags)?

**Reply by FFaranna** [reply]
I was thinking about it right now, and I guess with tags, it would be the most flexible way.
